test/system/events_test.rb

```diff
@@ -0,0 +1,9 @@
+require "application_system_test_case"
+
+class EventsTest < ApplicationSystemTestCase
+  # test "visiting the index" do
+  #   visit events_url
+  
+  #   assert_selector "h1", text: "Events"
+  # end
```

Comment:
_nitpick_ _suggestion_

Please remove this file. 
- Currently we don't display events on the website
- In general we should not have a commented test as in case of a refactoring this will just add an extra action without any benefit for the developer